.gitignore

```diff
@@ -61,6 +61,7 @@ Tests/Pcap++Test/cUrl/curl_output.txt
 *.VC.db
 *.vcxproj.user
 **/.vs
+CMakeSettings.json
```

Comment:
This is Visual Studio generated file to hold configuration when remote compilation (ssh/wsl etc) requested